packages/suite/src/components/suite/Modal/index.tsx

```diff
@@ -1,13 +1,6 @@
 import React from 'react';
 import { Modal as TrezorModal, ModalProps } from '@trezor/components';
-import { DESKTOP_WRAPPER_BORDER_WIDTH } from '@suite-constants/layout';
-import { isDesktop, isMacOs } from '@suite-utils/env';
 
-const Modal = (props: ModalProps) => (
-    <TrezorModal
-        {...props}
-        desktopBorder={isDesktop() && !isMacOs() ? DESKTOP_WRAPPER_BORDER_WIDTH : undefined}
-    />
-);
+const Modal = (props: ModalProps) => <TrezorModal {...props} />;
```

Comment:
This component became useless and could be replaced directly by Modal from `@trezor/components`.